docs/converters/converter.md

```diff
@@ -58,3 +58,23 @@ You can also add additional information about each component in the form of `ext
 ```python
 converter.save(example_data, extra_info=example_extra_info, destination=destination_path)
 ```
+
+## Configuring The Log Output
+ 
+We have provided an interface to configure the log level of the converter.
+Notice that this log level only belongs to the logger within the converter.
+Users need to set their basic configuration of the `logging` module to a level that is below what is configuired for the converters.
+ 
+```python
+# This is an example in your script
+import logging
+from power_grid_model_io.converters import VisionExcelConverter
+ 
+logging.basicConfig(level=logging.INFO) # Only levels INFO and above will be logged
+ 
+converter_warning = VisionExcelConverter(input_file, log_level=logging.WARNING) # If there is any logs above WARNING, they will be logged
```

Comment:
it's probably good to mention that this does not affect the `logging.basicConfig` or any other converters created after that. maybe even provide an example of that, e.g.

```suggestion
converter_warning = VisionExcelConverter(input_file, log_level=logging.WARNING) # If there is any logs above WARNING, they will be logged
converter_info = VisionExcelConverter(input_file) # uses default INFO level
```

Reply:
I think the line below did just that?